docs/register/prop-method-injection.rst

```diff
@@ -43,12 +154,45 @@ If the component is a :ref:`reflection component <register-registration-reflecti
     // is important!
     builder.RegisterType<C>().PropertiesAutowired(new MyCustomPropSelector());
 
+.. note::
+
+  Properties set on a component because they are ``required`` may lead to duplicate injection of a property if ``PropertiesAutowired`` is also used on that component.  Consider using ``PropertiesAutowired(PropertyWiringOptions.PreserveSetValues)`` to avoid repeating property injection.
```

Comment:
I didn't even think about this. Might be worth, at some point, figuring out how to track this so folks don't have to worry about it.

Reply:
Hmm, you're right...might actually make the change pre release to ignore required properties in the classic PropertiesAutowired, might as well fix it now.

Reply:
Seems reasonable.

Reply:
Looks like this got fixed up in the code.